hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterClockImpl.java

```diff
@@ -24,6 +25,8 @@
 public class ClusterClockImpl implements ClusterClock {
 
     private final ILogger logger;
+
+    @Probe(name = "clusterTimeDiff")
```

Comment:
This probe, I guess, should be places on `getClusterTimeDiff()` method instead of field.